modules/openapi-generator/src/main/resources/typescript/model/ObjectSerializer.mustache

```diff
@@ -110,12 +110,7 @@ export class ObjectSerializer {
             return transformedData;
         } else if (type === "Date") {
             if (format == "date") {
-                let month = data.getMonth()+1
-                month = month < 10 ? "0" + month.toString() : month.toString()
-                let day = data.getDate();
-                day = day < 10 ? "0" + day.toString() : day.toString();
-
-                return data.getFullYear() + "-" + month + "-" + day;
+                return data.toISOString().substring(0,10);
```

Comment:
I am not sure about this change, IIRC this caused troubles in the past when someone uses a local date and expects the local representation to be transferred over the wire, as opposed to a full iso string where the time zone information is there.

@TiFu any opinion here?

Reply:
@macjohnny, thank you for sharing your concern. While I'm confident that the dates should be transferred between the client and server with the UTC timezone, the main problem that I explained in https://github.com/OpenAPITools/openapi-generator/issues/17272 is that the `serialize` and `deserialize` methods are incompatible. This is because the current `deserialize` method implementation expects the date with the UTC timezone.

Reply:
@TiFu @macjohnny, hi guys. Do you have any updates on this issue?

Reply:
@akehir @amakhrov what is your take on this?